staging/src/k8s.io/apiserver/pkg/admission/configuration/validating_webhook_manager.go

```diff
@@ -61,8 +62,8 @@ func NewValidatingWebhookConfigurationManager(f informers.SharedInformerFactory)
 }
 
 // Webhooks returns the merged ValidatingWebhookConfiguration.
-func (v *validatingWebhookConfigurationManager) Webhooks() []v1beta1.Webhook {
-	return v.configuration.Load().(*v1beta1.ValidatingWebhookConfiguration).Webhooks
+func (v *validatingWebhookConfigurationManager) Webhooks() []webhook.WebhookAccessor {
+	return webhook.ConvertToValidatingWebhookAccessors(v.configuration.Load().(*v1beta1.ValidatingWebhookConfiguration).Webhooks)
```

Comment:
before merge, this conversion will need to move to updateConfiguration where we store the items, so we only convert the list when a webhook config object changes, instead of on every API call

Reply:
Done, thanks!